bin/migrate-oats-data/sql/insert-batch-application.sql

```diff
@@ -45,11 +45,19 @@ alcs_gov as(
     alg.uuid AS gov_uuid
 
     FROM oats_gov
-        JOIN alcs.application_local_government alg ON oats_gov.oats_gov_name = alg."name"
+        join alcs.application_local_government alg on
+   (case
+   	when oats_gov.oats_gov_name LIKE 'Islands Trust%' then 'Islands Trust'
+   	else oats_gov.oats_gov_name
+   end) 
+   =
+   (case
+   	when alg."name" LIKE 'Islands Trust%' then 'Islands Trust'
```

Comment:
I thought that alcs.application_local_government has only 'Islands Trust' and no variations. If that is the case then the right 'case when' is redundant

Reply:
It definitely is